featurebyte/sql/bigquery/F_COUNT_DICT_LEAST_FREQUENT.sql

```diff
@@ -0,0 +1,5 @@
+CREATE OR REPLACE FUNCTION `{project}.{dataset}.F_COUNT_DICT_LEAST_FREQUENT`(counts JSON)
+  RETURNS STRING
+  AS (
+    `{project}.{dataset}.F_COUNT_DICT_MOST_FREQUENT_KEY_VALUE`(counts, true).key
+  );
```

Comment:
Seems like the implementation here is a bit different from the snowflake one:
```
CREATE OR REPLACE FUNCTION F_COUNT_DICT_LEAST_FREQUENT(counts variant)
  RETURNS string
  AS
$$
  CAST(GET(F_COUNT_DICT_MOST_FREQUENT_KEY_VALUE(counts, true), 0) AS string)
$$
;
```
Just want to check whether this is expected.

Reply:
Yes, that is expected because the `F_COUNT_DICT_MOST_FREQUENT_KEY_VALUE` returns a named struct in BigQuery instead. Though I think it's also possible make it return an array like in Snowflake so the implementations can match.